types/wordpress__blocks/api/registration.d.ts

```diff
@@ -167,3 +176,29 @@ export function unregisterBlockStyle(blockName: string, styleVariationName: stri
  * @returns The previous block value if successfully unregistered, otherwise `undefined`.
  */
 export function unregisterBlockType(name: string): Block<any> | undefined;
+
+/**
+ * Returns an array with the variations of a given block type.
+ *
+ * @param blockName - Name of block (example: “core/columns”).
+ * @param [scope] - Block variation scope name.
```

Comment:
should we be leaving the `[]` in here? I thought we'd rather rely on the `scope?:` part of the type annotation so that the comment remains purely about the description while the types are linked to the actual annotations which TypeScript enforces…